kubernetes/loculus/values.schema.json

```diff
@@ -245,14 +245,65 @@
                   "type": "boolean",
                   "default": true,
                   "description": "If false, the submission form will not allow submission of consensus sequences (i.e. the sequences file must be omitted). All consensus sequence related parts on the website will be hidden."
+                },
+                "files": {
+                  "type": "object",
+                  "properties": {
+                    "enabled": {
+                      "type": "boolean",
+                      "description": "TODO"
```

Comment:
Should we make issues for these?

Reply:
Added the docs, previewable here: https://docs-file-sharing.loculus.org/reference/helm-chart-config/#organism-schema-type